### ISSUE TITLE
Holos lose their scale on lag

### ISSUE THREAD

**Issue by shadowscion** [INPStarfall/Starfall#37]
When a lag spike occurs, holograms revert to their original scale.

You can force the issue by typing "record 1;stop" in console.

**Comment by shadowscion** [comment]
Related issue; holograms are unscaled and unclipped for players joining after the holograms were spawned.

**Comment by Xandaros** [comment]
Yeah, I can imagine how the second bugs works. (In fact, I had identified it already, but forgot to open an issue. Shame on me) I have no idea how lag can cause such behaviour, though...

**Comment by shadowscion** [comment]
E2 used to do it aswell; here is how they fixed it:

https://github.com/wiremod/wire/commit/b441edff1dad9a819d512bcf2dff17f44403d247

**Comment by ColonelThirtyTwo** [comment]
When a player's connection experiences a large ping/packetloss spike, the source engine sends a full state update. This probably involves re-creating entities, which would explain why some SENTs break and you hear the weapon pickup noise when you get your connection back.
